Source/WebKit/Shared/WebCoreArgumentCoders.h

```diff
@@ -144,6 +144,7 @@ class FragmentedSharedBuffer;
 class StickyPositionViewportConstraints;
 class SystemImage;
 class TextCheckingRequestData;
+class TransformOperation;
```

Comment:
Looks like this was not enough due to the use of WebCore::TransformOperation::Type. You'll probably need to include the header? That should fix the WPE EWS.